src/_damon_args.py

```diff
@@ -265,8 +265,21 @@ def damos_options_to_schemes(args):
             nr_schemes - len(args.damos_apply_interval))
     args.damos_quotas += [None] * (nr_schemes - len(args.damos_quotas))
     args.damos_wmarks += [None] * (nr_schemes - len(args.damos_wmarks))
+    target_nid = [None] * nr_schemes
     schemes = []
+
     for i in range(nr_schemes):
+        action = args.damos_action[i][0]
```

Comment:
This may not be valid when the action is one of non-migrate actions.

Reply:
Even for non-migrate actions, the list (`args.damos_action[i]`) would have at least one item, right?  So I'm not sure what real problem could be caused.

Reply:
Right. It's not a problem.